packages/controller-utils/src/types.ts

```diff
@@ -1,13 +1,31 @@
 /**
- * Human-readable network name
+ * Ethereum networks that can be used to create an Infura network client.
  */
-export enum NetworkType {
-  localhost = 'localhost',
-  mainnet = 'mainnet',
-  goerli = 'goerli',
-  sepolia = 'sepolia',
-  rpc = 'rpc',
-}
+export const InfuraNetworkType = {
+  mainnet: 'mainnet',
+  goerli: 'goerli',
+  sepolia: 'sepolia',
+} as const;
+
+/**
+ * Ethereum networks that can be used to create an Infura network client.
+ */
+export type InfuraNetworkType =
```

Comment:
It is unfortunate that this type is named the same as the value to which it refers. It means that when importing this type, you must use `import type` (otherwise if you just use `import` you'd get the value). To be fair, we ought to be using `import type` for types anyway, so that's a better practice, but it may be confusing. I opted to do this for backward compatibility, but we can improve this in the future if we like.

Reply:
If you import it without using `type`, you should get the value _and_ the type. This should behave exactly like an enum. I'm not sure why we'd consider this unfortunate.

Reply:
Hmm, I was having trouble with getting this to work correctly when testing just `import` as opposed to `import type`, but now that I've tried this again, it does seem to work as you say.